lms/templates/registration/account_activation_sidebar_notice.html

```diff
@@ -0,0 +1,34 @@
+<%page expression_filter="h"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="profile-sidebar" role="region" aria-label="Account Activation Info">
+  <header class="profile">
+    <h2 class="account-activation-title sr">${_("Account Activation Info")}: </h2>
+  </header>
+  <div class="user-info">
+    <ul>
+      <li class="status status-verification warning" role="alert" aria-labelledby="title status-title" tabindex="1">
```

Comment:
Yikes!  This merged?  You cannot use class names as values for aria-labelledby.  That attribute only accepts IDrefs.  Also, unless the text content of this node is added AFTER DOM load, a role=alert won't do anything.  Role=alert is for dynamically adding an alert to a page that has already been loaded.  If it loads like this, then nothing is changing, and the alert status will never be triggered.